src/components/shared/cellRenderers/isSimulatedRenderer/isSimulatedRenderer.js

```diff
@@ -7,11 +7,11 @@ import { svgs } from 'utilities';
 
 import '../cellRenderer.css'
 
-export const IsSimulatedRenderer = ({ value }) => (
+export const IsSimulatedRenderer = ({ value, context: { t } }) => (
   <div className="pcs-renderer-cell highlight">
-    <Svg path={value ? svgs.simulatedDevice : svgs.physicalDevice} className="pcs-renderer-icon" />
-    <div className="pcs-renderer-text">
-      { value ? 'YES' : 'NO' }
+    <Svg path={ value ? svgs.simulatedDevice : svgs.physicalDevice } className="pcs-renderer-icon" />
```

Comment:
Not particularly relevant to this review:  But is there a legend somewhere in the UI for these symbols? or in the help?

Reply:
Nope